Makefile

```diff
@@ -11,6 +11,15 @@ export JAEGER_SAMPLER_TYPE:=const
 export JAEGER_SAMPLER_PARAM:=1
 export LOG_FORMAT:=console
 
+test:
+	go test -mod vendor -count=1 -v -cover -race -short ./...
+
+integration:
+	go test -mod vendor -count=1 -v -cover -race -run TestIntegration ./...
+
+ci:
```

Comment:
Add `ci` to `.PHONY` and move `install` back to the top so that it stays the default make task.

Reply:
done